.github/workflow_scripts/e2e_cpu_check.sh

```diff
@@ -0,0 +1,13 @@
+# Move to parent directory
+cd ../../
+
+set -ex
+
+sh ./tests/end2end-tests/setup.sh
+sh ./tests/end2end-tests/create_data.sh
+sh ./tests/end2end-tests/tools/test_mem_est.sh
```

Comment:
This is not needed.

Reply:
we would need _create_data.sh_ for partitioning

Reply:
I mean we do not need `sh ./tests/end2end-tests/tools/test_mem_est.sh`

Reply:
Done